packages/keystone/lib/ListTypes/list.js

```diff
@@ -1483,18 +1489,33 @@ module.exports = class List {
         getDelete: () => context.getListAccessControlForUser(this.key, undefined, 'delete'),
         getAuth: () => context.getAuthAccessControlForUser(this.key),
       }),
+
       getSchema: () => {
-        const queries = [
-          this.gqlNames.itemQueryName,
-          this.gqlNames.listQueryName,
-          this.gqlNames.listQueryMetaName,
-        ];
+        const queries = {
+          item: this.gqlNames.itemQueryName,
+          list: this.gqlNames.listQueryName,
+          meta: this.gqlNames.listQueryMetaName,
+        };
```

Comment:
What's the impact of changing the shape of this? I'm not up to speed on how the `getSchema ` method and `queries` are used. Also adding `mutations` I assume is just for consumption by developers? 

@timleslie assuming my reading is correct are you ok with exposing this API?

Reply:
`getSchema` helps populates the `schema` field in the `_ListMeta` type. I don't think `queries` is used for anything yet. Basically, there needs to be some way to access query/mutation names in the admin UI to dispatch the appropriate operations. Right now, the whole `gqlNames` object is passed through in the JSON global.